test/nn/test_convolution.py

```diff
@@ -481,6 +481,7 @@ def test_ConvTranspose2d_half_cublas_gemm(self):
     # For https://github.com/pytorch/pytorch/pull/1273
     # Almost identical to the above `test_Conv2d_naive_groups`
     @torch.backends.cudnn.flags(enabled=True, benchmark=False)
+    @unittest.skipIf(TEST_WITH_ROCM, "Skipped on ROCm")
```

Comment:
Better to update the skip reason saying "Skipped on ROCm, since it is failing on ROCm 5.7"